contribute/design/1.x/cstor/zcmd-operation.md

```diff
@@ -0,0 +1,232 @@
+---
+oep-number: ZCommands Execution Via CSP Custom Resource
+title: ZCommands execution via custom resources
+authors:
+  - "@mittachaitu"
+owners:
+  - "@kmova"
+editor: "@mittachaitu"
+creation-date: 2020-08-26
+last-updated: 2020-08-26
+status: provisional
+---
+
+# ZCommands execution via Custom Resource
+
+## Table of Contents
+
+- [ZCommands execution via Custom Resource](#zcommands-execution-via-custom-resource)
+	- [Table of Contents](#table-of-contents)
+	- [Summary](#summary)
+	- [Motivation](#motivation)
+		- [Goals](#goals)
+	- [Proposal](#proposal)
+		- [Stories](#stories)
+		- [Proposed Implementation](#proposed-implementation)
+		- [Steps to be performed by high level operator](#steps-to-be-performed-by-high-level-operator)
+		- [Low Level Design](#low-level-design)
+			- [WrokFlow](#wrokflow)
+
+## Summary
+
+This proposal brings out the design details to implement zpool/zfs commands execution
+via CSP custom resource.
+
+## Motivation
+
+- Zpool and ZFS command execution via CSP custom resource
+
+### Goals
+
+- High level operators which requires execution of Zpool/ZFS command can be executed via CStorPool(CSP) custom resource.
+
+## Proposal
+
+### Stories
+
+High level operator should able to execute Zpool/ZFS commands even if user has tighten the RBAC rules.
```

Comment:
tighten RBAC rules ?? Which component is impacted and how due to this?

Reply:
Also, how a high-level operator executes zpool/zfs command ?? It is always the cstor-pool-mgmt. 
Are we planning to have any operator that will run as a sidecar like cstor-pool-mgmt?

Reply:
> how a high-level operator executes zpool/zfs command ??

There is a section called #steps-to-be-performed-by-high-level-operator which will address How part.